src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenceHistoryDetailDto.kt

```diff
@@ -35,5 +35,7 @@ data class OffenceHistoryDetailDto(
     this.courtDate,
     this.caseId,
     null,
+    null,
```

Comment:
this is when mapped outside of JPA, so by setting to `null` the fields won't be returned i.e. no change to other endpoints

Reply:
although I guess they will still be in the swagger docs

Reply:
I think that is fine, I have done that and just added a note to swagger to explain